person.rb

```diff
@@ -0,0 +1,38 @@
+require './nameable'
+require './decorators'
+
+# Person Class inherits from Nameable
+class Person < Nameable
```

Comment:
- [ ] Again, we need to explicitly define a method for adding rentals in _Person_ class for this activity. This should be similar to:
```ruby
class Person
  attr_accessor :name, :age, :rentals
  .
  .

  def initialize(age, name = "Unknown", parent_permission = true)
    .
    .
    @rentals = []
  end

  def add_rental(book, date)

    Rental.new(date, book, self)
  end

  .
  .
end
```